flink-core/src/main/java/org/apache/flink/configuration/DeploymentOptions.java

```diff
@@ -119,9 +119,8 @@ public class DeploymentOptions {
                                     .linebreak()
                                     .linebreak()
                                     .text(
-                                            "Currently, this list is limited to '%s' only.",
-                                            TextElement.text(
-                                                    PipelineOptions.GLOBAL_JOB_PARAMETERS.key()))
+                                            "Currently changes to the ExecutionConfig class, that are not backed by the"
```

Comment:
Phrase this more generic because we now also support `CheckpointCfg`: "...that are not backed by config options."